widgetbook_for_widgetbook/lib/navigation/navigation_tree_item.dart

```diff
@@ -55,12 +56,14 @@ Widget navigationTreeItemWithout(BuildContext context) {
         isExpanded: context.knobs.boolean(
           label: 'Is Expanded',
           description: 'Only '
+              // ignore:lines_longer_than_80_chars
```

Comment:
Is this necessary? 😀 We can make this line run over multiple lines.

Reply:
Fixed